templates/contact.html

```diff
@@ -8,40 +8,43 @@
 <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"></script>
 {% endblock head %}
 
-{% macro row(title, slug, email, people) %}
-<tr>
-  <td> <!--class="d-flex align-items-center gap-2">-->
-    <div class="fs-4 icon baseline"><img src="{{ get_url(path="assets/exec-icons/" ~ slug ~ ".svg") }}"></div>
-    {{ title }}
-  </td>
-  <td>
-    {{ people | join(sep=", ") }}
-  </td>
-  <td>
-      <a target="_blank" rel="noopener" href="mailto:{{ email }}"
-        aria-label="Email our {{ title }}">{{ email }}</a>
-  </td>
-</tr>
-{% endmacro %}
-
 {% block content %}
 
 {{ resource.content | safe}}
 
-<table class="table table-striped exec-table">
-  <thead>
-    <tr>
-      <th>Role</th>
-      <th>People</th>
-      <th>Contact</th>
-    </tr>
-  </thead>
-  <tbody>
-      {{ self::row(title="Exec", slug="all", email="exec@uwcs.co.uk", people=["All Exec"]) }}
-    {% for role in section.pages %}
-      {{ self::row(title=role.title, slug=role.slug, email=role.extra.email, people=role.extra.people) }}
-    {% endfor %}
-  </tbody>
-</table>
+{% for role in section.pages %}
+  <div class="role">
+    <div class="fs-4 icon baseline exec-title">
+      <img src="{{ get_url(path='assets/exec-icons/' ~ role.slug ~ '.svg') }}" alt="{{ role.title }}">
+      {% if role.extra.people | length > 1 %}
+        <h3>{{ role.extra.plural }}<h3>
+      {% else %}
+        <h3>{{ role.title }}<h3>
+      {% endif %}
+    </div>
+    <a href="mailto:{{role.extra.email}}" aria-label="Email our {{role.title}}">{{role.extra.email}}</a>
+  </div>
+{% endfor %}
+<div class="exec-cards">
+{% set harmony = true %}
+{% for role in section.pages %}
+  {% for person in role.extra.people %}
+    {# because Harmony just has to be special they have 2 roles #}
+    {% if person == "Harmony Bouabid" and harmony %}{% set_global harmony = false %}{% continue %}{% endif %}
+    <div class="card exec-card">
+      {# for some godforsaken reason, you cannot concatonate more than 3 strings in zola #}
+      {% set path = "assets/exec-pics/" ~ person | replace(from=" ",to="") | lower %}
+      {% set path = path ~ ".jpg" %}
+      {# the linting on this line is some of the worst i have ever seen #}
+      <img src="{{ get_url(path=path) }}" alt="{{ person }}" class="card-image-top" onerror="this.src='{{get_url(path='assets/exec-pics/default.png')}}';">
+      <div class="card-body">
+        <h4 class="card-title">{{ person }}</h4>
+        <p class="card-text">{{ role.title }}</p>
+        {% if person == "Harmony Bouabid" %}<p class="card-text">Inclusivity Officer</p>{% endif %}
```

Comment:
IMO, instead of listing both roles separately, have Harmony's as "Publicity & Inclusivity Officer". If you don't want to do that, at least put the two roles in the same \<p> with a \<br /> between them, so there isn't the massive gap between them